apps/base-docs/tutorials/docs/1_verify-contract-with-basescan.md

```diff
@@ -0,0 +1,291 @@
+
+---
+
+## Objectives
+
+By the end of this tutorial, you should be able to:
+
+- Deploy a smart contract using [Foundry]
+- Interact with the [Basescan API] to verify your deployed contract
+- Obtain and configure a (free) Base RPC Node from [Coinbase Developer Platform (CDP)](https://portal.cdp.coinbase.com/products/node)
+
+---
+
+## Prerequisites
+
+**Familiarity with smart contract development and the Solidity programming language**
+
+Solidity is the primary programming language for writing smart contracts on Ethereum and Ethereum-compatible blockchains like Base. You should be comfortable with writing, compiling, and deploying basic smart contracts using Solidity.
+
+**Basic understanding of Foundry for Ethereum development**
+
+Foundry is a fast and portable toolkit for Ethereum application development. It simplifies the process of deploying, testing, and interacting with smart contracts. This tutorial assumes you have experience using Foundry to compile and deploy smart contracts.
+
+**Access to a Coinbase Developer Platform (CDP) account**
+
+The [Coinbase Developer Platform] provides access to tools and services necessary for blockchain development, including RPC nodes for different networks. You'll need to sign up for a CDP account to obtain a [Base RPC Node], which will be essential for deploying and interacting with your smart contracts on the Base blockchain.
+
+**Node + Basic API requests**
+
+## Jump Right In
+
+For this turotial, we will deploy a simple contract that is included in the Foundry quickstart. To do so, ensure that you have Foundry installed.
+
+If you don't have Foundry install it:
+
+```bash
+curl -L https://foundry.paradigm.xyz | bash
+```
+
+Once installed, create a Foundry project:
+
+```bash
+forge init verify_contracts
+```
+
+then change into the newly made directory:
+
+```bash
+cd verify_contracts
+```
+
+You should have a folder structure similar to this:
+
+```bash
+├── lib
+├── script
+├── src
+└── test
+```
+
+The `src` folder will contain a `Counter.sol` file which will serve as the country we want to deploy.
+
+:::note You will need ETH on Base to deploy
+You (the deployer wallet) will need some ETH in order to broadcast the transaction to the Base network. Fortunately, transactions are usually < 1 cent on Base mainnet.
+
+If using a [Coinbase Wallet] use the "Buy" button to onramp crypto from your Coinbase account.
+:::
+
+You will need a private key of the wallet that you want to deploy the smart contract from. Obtain it and store it as an env variable in your terminal.
+
+Once you have the private key to the wallet of your choice, open your terminal and store it in an environment variable:
+
+```bash
+export PRIVATE_KEY="f980byuhjhgfr45tyhglQxydfG5yu765rf"
```

Comment:
I'd replace this with <your key here> 

Otherwise, I worry about someone accidentally doing something important with an exposed key.